oap-server/server-query-plugin/mqe-grammar/src/main/antlr4/org/apache/skywalking/mqe/rt/grammar/MQEParser.g4

```diff
@@ -27,7 +27,7 @@ expression
     | expression mulDivMod expression  # mulDivModOp
     | expression addSub expression     # addSubOp
     | expression compare expression    # compareOp
-    | aggregation L_PAREN expression R_PAREN # aggregationOp
+    | aggregation L_PAREN expression (COMMA parameter)* R_PAREN # aggregationOp
```

Comment:
It's better to create a new expression like `top_n` or `relabels`. An expression should have determined parameters to avoid unexpected parsing.